askama_parser/src/tests.rs

```diff
@@ -498,109 +498,109 @@ fn test_odd_calls() {
 fn test_parse_comments() {
     let s = &Syntax::default();
 
+    trait OneCommentWs {
+        fn one_comment_ws(self) -> Ws;
+    }
+
+    impl OneCommentWs for Vec<Node<'_>> {
+        fn one_comment_ws(mut self) -> Ws {
+            assert_eq!(self.len(), 1);
+            match self.pop().unwrap() {
+                Node::Comment(comment) => comment.ws,
+                val => panic!("expected comment, but parsed={:?}", val),
+            }
+        }
+    }
+
     assert_eq!(
-        Ast::from_str("{##}", s).unwrap().nodes,
-        vec![Node::Comment(Ws(None, None))],
+        Ast::from_str("{##}", s).unwrap().nodes.one_comment_ws(),
```

Comment:
I feel like `one_comment_ws()` should just be a function that takes a `&str`, so it can subsume the whole thing starting from `Ast::from_str()`. Any reason not to do so?